PyInstaller/hooks/hook-usb.py

```diff
@@ -23,37 +21,53 @@
 # https://github.com/walac/pyusb/blob/master/docs/faq.rst
 # https://github.com/walac/pyusb/blob/master/docs/tutorial.rst
 
-binaries=[]
+binaries = []
 
-# first try to use pyusb library locator
-try:
-    # get the backend symbols before find
-    pyusb_backend_dir = set(dir(usb.backend))
 
-    # perform find, which will load a usb library if found
-    usb.core.find()
+# special-case cygwin, try to use the explicit cygwin usb dlls.
+if is_cygwin:
+    bins = ['cygusb-1.0-0.dll', 'cygusb0.dll']
+    binaries = _resolveCtypesImports(bins)
 
-    # get the backend symbols which have been added (loaded)
-    backends = set(dir(usb.backend)) - pyusb_backend_dir
 
-    # for each of the loaded backends, see if they have a library
-    binaries = []
-    for usblib in [getattr(usb.backend, be)._lib for be in backends]:
-        if usblib is not None:
-            binaries = _resolveCtypesImports([os.path.basename(usblib._name)])
-            assert len(binaries[0]) == 3
-            binaries = [(binaries[0][1], '')]
+# loading usb.core.find in this script crashes Ubuntu 14.04LTS,
+# let users circumvent pyusb discovery with an environment variable.
+skip_pyusb_discovery = \
+    getenv('PYINSTALLER_USB_HOOK_SKIP_PYUSB_DISCOVERY') is not None
```

Comment:
1) Setting en environment variable to an empty string (`FOO=`) instead of unsetting it (`unset FOO`) thus I suggest do simply use
`skip_pyusb_discovery = bool(getenv('PYINSTALLER_USB_HOOK_SKIP_PYUSB_DISCOVERY'))`
What do you think?

2) How does this "crash" show up? Can we catch an exception in that case and print a message telling to set the env-var? This trick is not documented anywhere.

3) Anyway please put this into a commit of its own.